src/perfcollect/perfcollect

```diff
@@ -1765,16 +1814,21 @@ DoCollect()
     then
         WriteStatus "Collection started. Collection will automatically stop in $duration second(s).  Press CTRL+C to stop early."
     else
-            WriteStatus "Collection started. Press CTRL+C to stop."
+        WriteStatus "Collection started."
```

Comment:
Please make sure to leave the CTRL+C message here if the user runs perfcollect collect so that they know what to do to stop collection.